Dockerfile

```diff
@@ -64,18 +78,31 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 # Support Gradle
 ENV TERM dumb
 ENV JAVA_OPTS "-Xms512m -Xmx1024m"
```

Comment:
The issue is.. this container is not just used by Uber after it's now open source'd. I think that setting can be moved into the inherited image with customized changes.  In fact, you raise a good point, maybe we should removed these hard-coded memory settings too.

You can always run the build in this way:

`./gradlew -Dorg.gradle.jvmargs="-Xms1g -Xmx4g" clean assemble`